.golangci.yml

```diff
@@ -35,7 +35,7 @@ linters:
     - containedctx
     # - contextcheck
     - decorder
-    - depguard
+    # - depguard
```

Comment:
depguard starting in 1.53.0 prevents importing anything other than the stdlib by default. If we want to configure this linter, we should do so in a follow up.